### DISCUSSION TITLE
Does anyone know why sdrpp looks like this on my laptop?

### DISCUSSION THREAD

**Discussion by bnelson333** [AlexandreRouma/SDRPlusPlus#673] [Q&A] [answered]
I love sdrpp, it's so much more stable than that other one out there. When I use it on my desktop, it works just fine, no problems. But when I run it on my laptop, visually, it looks really off. It almost looks like an overlay that I need to dismiss or something. See attached, any ideas? I'd really love to be able to use this on my laptop, which is a Dell E6410, rtl dongles not plugged in directly, I access them via rtl_tcp running on a separate raspberry pi. Again works just fine on my desktop.

![image](https://user-images.githubusercontent.com/12589803/155887680-6712de88-2379-4bdc-83c4-b3aec460f43d.png)

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
This is likely an issue with your graphics driver, check if you can update it. Also see #613 for a similar issue.

**Reply by bnelson333** [reply]
Thanks, I believe you are correct. I installed the latest video drivers and it made a difference. Didn't work, now it's all blue instead, but that leads me to believe it's driver related and this machine is probably too old to handle it. Thanks!
